src/components/SameStyleProducts/index.module.scss

```diff
@@ -0,0 +1,21 @@
+@import 'styles/constants';
+
+.wrapper {
```

Comment:
here I can see the same styles as for `VisitedProducts`

combine these two components